config/puma.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env puma
+
+rackup DefaultRackup
+
+environment ENV['RACK_ENV'] || 'development'
```

Comment:
Stylistically I think we should favour `fetch` over booleans when coalescing. Any thoughts?